### PR TITLE
Change link in tutorial from master to main

### LINKED ISSUE #209: Urls for datasets in Prerequisites does not exist anymore
https://github.com/ESMValGroup/ESMValTool_Tutorial/raw/master/data/dataset.urls
Datasets used for running examples/recipe_python.yml in episode 4 is not available (with the path above).
I tried with this urls : https://github.com/ESMValGroup/ESMValTool_Tutorial/blob/main/data/dataset.urls, but it seems not to be the proper datasets either.

### PR DESCRIPTION
**Pull request by Peter9192**
Because the master branch was renamed to main.

Closes #209

### CODE REVIEW

**Review by antoine1501 — CHANGES_REQUESTED** [review]
I would rather write this link (blob/ rather than raw/)
--> https://github.com/ESMValGroup/ESMValTool_Tutorial/blob/main/data/dataset.urls
Sorry for the duplication of modification request for this.

File: setup.md [hunk 1]
```diff
@@ -281,4 +281,4 @@ ESMValTool development team.
 
 [cmip5-search]: https://esgf-data.dkrz.de/search/cmip5-dkrz/
 [theatoga.nc]: http://esgf-data1.ceda.ac.uk/thredds/fileServer/esg_dataroot/cmip5/output1/MOHC/HadGEM2-ES/historical/mon/ocean/Omon/r1i1p1/v20110916/thetaoga/thetaoga_Omon_HadGEM2-ES_historical_r1i1p1_185912-200512.nc
-[ds]: https://github.com/ESMValGroup/ESMValTool_Tutorial/raw/master/data/dataset.urls
\ No newline at end of file
+[ds]: https://github.com/ESMValGroup/ESMValTool_Tutorial/raw/main/data/dataset.urls
```
Comment (Peter9192) [inline setup.md:284]:
```suggestion
[ds]: https://github.com/ESMValGroup/ESMValTool_Tutorial/blob/main/data/dataset.urls
```

**Comment by Peter9192** [comment]
> 
> 
> I would rather write this link (blob/ rather than raw/)
> --> https://github.com/ESMValGroup/ESMValTool_Tutorial/blob/main/data/dataset.urls
> Sorry for the duplication of modification request for this.

Modified. Note that the other one that still has 'raw' needs that, otherwise the download command doesn't work.

**Review by antoine1501 — APPROVED** [review]
Link properly modified

**Comment by Peter9192** [comment]
@axel-lauer could you add @antoine1501 to the esmvaltool development team?

**Comment by Peter9192** [comment]
@rswamina could you approve the PR so we can merge it?

**Review by rswamina — APPROVED** [review]

**Comment by Peter9192** [comment]
Thanks!

**Pull request merged**